src/main/kotlin/no/nav/familie/ef/sak/brev/FritekstBrevController.kt

```diff
@@ -10,13 +9,13 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping(path = ["/api/manueltbrev"])
+@RequestMapping(path = ["/api/brev"])
 @ProtectedWithClaims(issuer = "azuread")
-class ManueltBrevController(private val brevClient: BrevClient) {
+class FritekstBrevController(private val frittståendeBrevService: FrittståendeBrevService) {
 
-    @PostMapping("")
-    fun lagManueltBrev(@RequestBody brevInnhold: JsonNode): Ressurs<ByteArray> {
-        return Ressurs.success(brevClient.lagManueltBrev(brevInnhold))
+    @PostMapping("/frittstaende")
```

Comment:
Hvis vi endrer base-pathen kan denne pathen fjernes
```suggestion
    @PostMapping
```

Reply:
Fikset i 2d7cca8